src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Views/Shared/DraftApprenticeshipOverlapOptions.cshtml

```diff
@@ -0,0 +1,97 @@
+﻿@using SFA.DAS.CommitmentsV2.Shared.Extensions
+@using SFA.DAS.ProviderCommitments.Web.Models
+@model SFA.DAS.ProviderCommitments.Web.Models.DraftApprenticeshipOverlapOptionViewModel
+@inject SFA.DAS.Authorization.Features.Services.IFeatureTogglesService<SFA.DAS.Authorization.ProviderFeatures.Models.ProviderFeatureToggle> featureToggleService
+
+@{
+    var overlappingToggleEmailRequest = featureToggleService.GetFeatureToggle(SFA.DAS.ProviderCommitments.Features.ProviderFeature.OverlappingTrainingDate).IsEnabled;
```

Comment:
can we set this on the property rather than injecting the service into the model, this makes it more testable